### ISSUE TITLE
Nothing gets created

### ISSUE THREAD

**Issue by bertt** [igd-geo/schwarzwald#3]
Hi,

I'm trying to create 3D Tiles from https://storage.sbg.cloud.ovh.net/v1/AUTH_63234f509d6048bca3c9fd7928720ca1/ppk-lidar/NP/LHD_FXX_0809_6305_PTS_O_LAMB93_IGN69.copc.laz , but nothing gets created any idea? Docker image is build from latest source code.

Command to run:

```
docker run -v $(pwd):/nimes -it schwarzwald:latest --tiler -i /nimes -o /nimes/output
```

Console output: 

```
warning: Not all point attributes in the input files are supported when using output format 3DTILES. Input files have attributes [ POSITION POINT_SOURCE_ID SCAN_ANGLE_RANK SCAN_DIRECTION_FLAG INTENSITY CLASSIFICATION GPS_TIME EDGE_OF_FLIGHT_LINE NUMBER_OF_RETURNS RETURN_NUMBER USER_DATA ], 3DTILES only supports attributes [ INTENSITY RGB POSITION ], so attributes [ USER_DATA RETURN_NUMBER NUMBER_OF_RETURNS SCAN_ANGLE_RANK CLASSIFICATION POINT_SOURCE_ID SCAN_DIRECTION_FLAG GPS_TIME EDGE_OF_FLIGHT_LINE ] will be ignored!
Writing the following point attributes: [ INTENSITY POSITION ]
Output directory does not exist, creating it
Total points: 18686454
Bounds:
min: [809000, 6304000, 32.35999999999999943] max: [810000, 6305000, 80.87999999999999545]
Bounds (cubic):
min: [809000, 6304000, -443.3799999999999955] max: [810000, 6305000, 556.6200000000000045]
Spacing calculated from diagonal: 6.9282
Using MIN_DISTANCE sampling
Input files:
"/nimes/LHD_FXX_0809_6305_PTS_O_LAMB93_IGN69.copc.laz"
[00:00:00] indexing: 0 / 18.686M loading: 0 / 18.686M
[00:00:05] indexing: 0 / 18.686M loading: 0 / 18.686M
```

**Comment by tobias93** [comment]
Hi, thanks for your feedback. I'll have a look on this in the next few days.

**Comment by tobias93** [comment]
Hi, this should be fixed with commit cc1a5c065f894cfac75652d55467db38d99053b1

**Comment by bertt** [comment]
Thanks, I can confirm pnts/json files are now created
